test/e2e/flask/create-watch-account.spec.ts

```diff
@@ -1,367 +0,0 @@
-import { strict as assert } from 'assert';
-import { Suite } from 'mocha';
-import FixtureBuilder from '../fixture-builder';
-import { withFixtures } from '../helpers';
-import { Driver } from '../webdriver/driver';
-import AccountDetailsModal from '../page-objects/pages/dialog/account-details-modal';
-import AccountListPage from '../page-objects/pages/account-list-page';
-import ExperimentalSettings from '../page-objects/pages/settings/experimental-settings';
-import HeaderNavbar from '../page-objects/pages/header-navbar';
-import HomePage from '../page-objects/pages/home/homepage';
-import SettingsPage from '../page-objects/pages/settings/settings-page';
-import { loginWithBalanceValidation } from '../page-objects/flows/login.flow';
-import { watchEoaAddress } from '../page-objects/flows/watch-account.flow';
-
-const ACCOUNT_1 = '0x5CfE73b6021E818B776b421B1c4Db2474086a7e1';
-const EOA_ADDRESS = '0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045';
-const SHORTENED_EOA_ADDRESS = '0xd8dA6...96045';
-const DEFAULT_WATCHED_ACCOUNT_NAME = 'Watched Account 1';
-
-describe('Account-watcher snap', function (this: Suite) {
```

Comment:
@davibroc we have a ticket to update this test so we should skip it instead of deleting the file. This is the ticket for it

https://github.com/MetaMask/metamask-extension/issues/37563